types/consumer-data-standards/README.md

```diff
@@ -0,0 +1,78 @@
+# CDS Type Definition Library
```

Comment:
The DT publisher ignores README.md and publishes its own. You can put this here but it won't be visible at (for example) https://www.npmjs.com/package/@types/aframe